update-imported-docs/reference.yml

```diff
@@ -6,10 +6,10 @@ repos:
   branch: master
   generate-command: |
     cd $GOPATH
-    git clone https://github.com/kubernetes/kubernetes.git src/k8s.io/kubernetes
+    git clone --depth=1 --single-branch --branch v${K8S_RELEASE}.0 https://github.com/kubernetes/kubernetes.git src/k8s.io/kubernetes
     cd src/k8s.io/kubernetes
     # set the branch, ex: v1.17.0
-    git checkout v${K8S_RELEASE}.0
+    # git checkout v${K8S_RELEASE}.0
```

Comment:
nit: It's probably tidier to omit this line.

Reply:
I have been moved the comment before the `git clone`, it seems more clear!
Thanks!

Reply:
Hello @Colstuwjx .
Could you remove line 12?

Reply:
Hi @Colstuwjx . I just checked this command and the branch does not exist (yet?).
```
Cloning into 'src/k8s.io/kubernetes'...
warning: Could not find remote branch v1.18.0 to clone.
```

Reply:
That's also I'm worrying about. Currently, k/k repo did have [v1.18.0-rc.1](https://github.com/kubernetes/kubernetes/releases/tag/v1.18.0-rc.1), rather than v1.18.0.

The PR would make the script ONLY clone the single branch, and it would fail if we need to checkout other branch.

Reply:
Right. You could leave this change as a comment and add a note about optimizing and assume that users know that the branch is not available until after the release is created?

Reply:
@kbhawkey I have been added more comments to clarify the risk, thanks for your suggestion!